<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>문제출제</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="qbackbutton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>490</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="text">
    <string>나가기</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="qtextBrowser">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>256</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="qtextBrowser_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>160</y>
     <width>256</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="qlineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>510</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="qpushbutton2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>490</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>출제</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
